synapse/storage/databases/main/devices.py

```diff
@@ -1529,50 +1529,53 @@ async def add_device_change_to_streams(
         if not device_ids:
             return None
 
-        async with self._device_list_id_gen.get_next_mult(
-            len(device_ids)
-        ) as stream_ids:
-            await self.db_pool.runInteraction(
-                "add_device_change_to_stream",
-                self._add_device_change_to_stream_txn,
+        num_stream_ids = max(
```

Comment:
it might be good to add a comment clarifying why this is a `max` rather than a total (vis, that we use the same set of stream ids for each of the three tables)

Reply:
The `max` got replaced, but I've added a comment the equivalent code